src/sdk/pynni/nni/batch_tuner/batch_tuner.py

```diff
@@ -42,11 +42,18 @@ class BatchTuner(Tuner):
                              '_value': '[{...}, {...}, {...}]',
```

Comment:
Currently The Search Space format cannot be well rendered under current numpydoc format. I suggest you either switch to an [Example](https://python-sprints.github.io/pandas/guide/pandas_docstring.html#section-7-examples) or put the search space specification in a separate document.

Reply:
Good suggestion. Changed.